backend/src/v4/routes/teamspace.js

```diff
@@ -547,18 +547,18 @@
 	}
 
 	function getMitigationsFile(req, res, next) {
-		TeamspaceSettings.getMitigationsStream(req.params.account).then((mitigations) => {
+		TeamspaceSettings.getMitigationsFile(req.params.account).then((mitigationsFile) => {
 			const timestamp = (new Date()).toLocaleString();
-			const filenamePrefix = (req.params.account + "_" + timestamp + "_").replace(/\W+/g, "_");
+			const filenamePrefix = (req.params.account + "_" + timestamp).replace(/\W+/g, "_");
 
 			const headers = {
 				"Content-Disposition": "attachment;filename=" + filenamePrefix + "mitigations.csv",
 				"Content-Type": "text/csv"
 			};
 
 			res.set(headers);
-
-			responseCodes.writeStreamRespond(utils.APIInfo(req), req, res, next, mitigations.readStream, headers);
+			res.send(mitigationsFile);
```

Comment:
Can we keep this as a streamed asset?

The backend is going to choke if we have a lot of rules.
Quick look at the docs it looks like the parser has a streaming api for this purpose 
![image](https://user-images.githubusercontent.com/11945337/152565500-6ef16b47-63f8-4a2d-9dcd-7b84b8dd1233.png)